packages/firebase_storage/firebase_storage_web/pubspec.yaml

```diff
@@ -8,24 +8,23 @@ environment:
   flutter: ">=1.20.0 <2.0.0"
 
 dependencies:
-  async: ^2.4.2
-  crypto: ^2.1.5
+  async: ^2.5.0-nullsafety.3
+  crypto: ^3.0.0-nullsafety.0
```

Comment:
the crypto null safe version here will cause this error:
```
Because every version of firebase_storage_web from path depends on crypto ^3.0.0-nullsafety.0 and every version of flutter_driver from sdk depends on crypto 2.1.5, firebase_storage_web from path is incompatible with flutter_driver from sdk.
So, because FlutterFire depends on both flutter_driver any from sdk and firebase_storage_web from path, version solving failed.
pub get failed (1; So, because FlutterFire depends on both flutter_driver any from sdk and firebase_storage_web from path, version solving failed.)
```
it clashes with `flutter_driver` crypto sub-dependency. melos only works when crypto is a dependency override. Not sure what to do if we're trying to avoid overrides.